pkg/okteto/preview.go

```diff
@@ -18,8 +18,8 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/machinebox/graphql"
 	"github.com/okteto/okteto/pkg/errors"
+	"github.com/shurcooL/graphql"
 )
 
 // DeployPreviewBody top body answer
```

Comment:
Can't `DeployPreviewBody`, `deprecatedDeployPreviewBody` and `PreviewBody` be deleted?